flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliStrings.java

```diff
@@ -353,13 +354,14 @@ public static AttributedString messageWarning(String message) {
                 .toAttributedString();
     }
 
-    public static AttributedString messageError(String message, Throwable t) {
+    public static AttributedString messageError(String message, Throwable t, boolean isVerbose) {
         while (t.getCause() != null
                 && t.getCause().getMessage() != null
                 && !t.getCause().getMessage().isEmpty()) {
             t = t.getCause();
         }
-        return messageError(message, t.getClass().getName() + ": " + t.getMessage());
+        String expMessage = isVerbose ? ExceptionUtils.getStackTrace(t) : t.getMessage();
```

Comment:
Flink has a util `org.apache.flink.util.ExceptionUtils#stringifyException` for this purpose, we should avoid to depend on other projects as much as possible.